src/routes/+page.svelte

```diff
@@ -17,6 +18,20 @@
 
 	<h2>Without title</h2>
 	<Youtube id="aYtE6XE6b_s" showTitle={false} />
+
+  <h2>With custom aria label button</h2>
+	<Youtube id="aYtE6XE6b_s" showTitle={false}>
+    <slot name="playButton">
+      <PlayButton ariaLabel="Custom play button" />
+    </slot>
```

Comment:
To make the examples work, slots must be used directly on the element

```suggestion
    <PlayButton slot="playButton" ariaLabel="Custom play button" />
```